### DISCUSSION TITLE
How to ingest cd/dvds and any recommendation for suitable format

### DISCUSSION THREAD

**Discussion by mobcdi** [gerbera/gerbera#1783] [Q&A] [answered]
Hi everyone,

I'm hoping to use gerbera as the basis for an audio library but i have a question about optical media usage and gerbera.

Is gerbera able to ingest media from an optical drive on the server itself, copy it to the catalog automatically or is it necessary to use another method to create source media on disk I then point gerbera to? I'd like to get the benefit of auto track naming while bulk importing/ingesting optical drives to create my catalog. Also is there a recommendation for a suitable file format to use for audio files that works best with gerbera and gives the best options for transcoding later on without having to re-rip/extract the audio files again? 
Thanks in advance for any help
Michael

**Comment by whyman** [comment, marked as answer]
> it necessary to use another method to create source media on disk I then point gerbera to

Yes, Gerbera cannot extract your audio. We expect media to be available on the filesystem

> Also is there a recommendation for a suitable file format to use for audio files that works best with gerbera and gives the best options for transcoding later on without having to re-rip/extract the audio files again?

A lossless format will preserve all the detail of the original, a lossless format that is well supported is FLAC.
However, FLAC is usually not well supported by clients, so you would need to set up transcoding to a more widely supported format, usually something like M4A or MP3.
